model_runner/validator/validator_utils.py

```diff
@@ -0,0 +1,126 @@
+        """
+        Validates if submitted gpu_type is currently available at Euler
+        (https://scicomp.ethz.ch/wiki/Getting_started_with_clusters#GPU).
+        """
+        gpu_list = [
+            "GeForceGTX1080",
+            "GeForceGTX1080Ti",
+            "GeForceRTX2080Ti",
+            "GeForceRTX2080Ti",
+            "TITANRTX",
+            "TeslaV100_SXM2_32GB",
+            "TeslaV100_SXM2_32GB",
+            "A100_PCIE_40GB",
+        ]
+
+        if v not in gpu_list:
+            raise ValueError(
+                f'"{v}" is currently not available on Euler. Check the list of available GPUs '
+                "at https://scicomp.ethz.ch/wiki/Getting_started_with_clusters#GPU."
+            )
+
+        return v
+
+    @validator("run_time")
+    def validate_and_coerce_run_time(cls, v):
+        """
+        Validates if submitted run_time adheres to the format specified in
+        https://scicomp.ethz.ch/wiki/Getting_started_with_clusters#Wall-clock_time.
+        """
+        if type(v) == str:
+            v_split = v.split(":")
+            if len(v_split) == 1:
+                try:
+                    minutes = int(v_split[0])
+                    v = minutes
+                except ValueError:
+                    raise ValueError(
+                        "run_time string must adhere to the format specified in "
+                        "https://scicomp.ethz.ch/wiki/Getting_started_with_clusters#Wall-clock_time."
+                    )
+            elif len(v_split) == 2:
+                try:
+                    minutes = int(v_split[1])
+                    if (minutes > 59) or (minutes < 0):
+                        raise ValueError(
+                            "run_time string must adhere to the format specified in "
+                            "https://scicomp.ethz.ch/wiki/Getting_started_with_clusters#Wall-clock_time."
+                        )
+
+                    hours = int(v_split[0])
+                    v = minutes + hours * 60
+                except ValueError:
+                    raise ValueError(
+                        "run_time string must adhere to the format specified in "
+                        "https://scicomp.ethz.ch/wiki/Getting_started_with_clusters#Wall-clock_time."
+                    )
+            else:
+                raise ValueError(
+                    "run_time string must adhere to the format specified in "
+                    "https://scicomp.ethz.ch/wiki/Getting_started_with_clusters#Wall-clock_time."
+                )
+
+        return v
+
+
+class ConfigModel(BaseModel):
+    data: str
+    runner: str
+    job_array: JobArrayModel
+    hyperparameters: Dict[str, List[Any]]
+
+    @validator("data")
+    def data_is_file_or_path(cls, v):
+        if not (os.path.isfile(v) or os.path.isdir(v)):
+            raise ValueError('"data" is not a file/directory.')
+        else:
+            if os.path.isdir(v) and v[-1] != "/":
+                return v + "/"
```

Comment:
I think it is better to get the separator from the `os.path` module so it is valid on all platforms.

```suggestion
                return v + os.path.sep
```